src/common/xo/create-network-modal/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react'
+
+import _, { messages } from '../../messages'
+import SingleLineRow from '../../single-line-row'
+import { SelectPif } from '../../select-objects'
+import { Col } from '../../grid'
+import { injectIntl } from 'react-intl'
+
+class CreateNetworkModalBody extends Component {
+  _host = this.props.container.type === 'pool' ? this.props.container.master : this.props.container.id
```

Comment:
This value is only used in _pifPredicate, do not add it to the current instance, it's not necessary.